packages/gestalt/src/TileData.js

```diff
@@ -0,0 +1,104 @@
+// @flow strict
+import { type Node } from 'react';
+import DataPoint from './Datapoint.js';
+import { useColorScheme } from './contexts/ColorSchemeProvider.js';
+import { type DataPointBaseProps } from './Datapoint/DataPointBaseProps.js';
+import Tile from './Tile/Tile.js';
+
+// Note: 03 is ommited because it doesn't have a corresponding dark token
+type DataVisualizationColors =
+  | '01'
+  | '02'
+  | '04'
+  | '05'
+  | '06'
+  | '07'
+  | '08'
+  | '09'
+  | '10'
+  | '11'
+  | '12';
+
+type Props = {|
+  ...DataPointBaseProps,
+  /**
+   * disables component interactivity
+   */
+  disabled?: boolean,
+  /**
+   * An identifier to be passed in a callback, and distinguish multiple DataPoints
```

Comment:
Which callback? Does this id need to be unique? If it's used for necessary disambiguation, why is it not required?